components/cloud_subscribe_result_modal/style.scss

```diff
@@ -0,0 +1,8 @@
+.IconMessage {
```

Comment:
@nickmisasi 
<img width="1792" alt="Screenshot 2022-07-05 at 12 16 19" src="https://user-images.githubusercontent.com/28563179/177294737-ce92bfff-ddde-4fe3-aae1-77b432dcab7e.png">

Could be breaking because of this? 🤔